File: HarfNotu.swift
//  Created by Kağan on 15.01.2017.
//  Copyright © 2017 Kağan Utku Kılıçlı. All rights reserved.

import Foundation

print("notunuzu giriniz:")
if var not = Int(readLine(strippingNewline: true)!) {
    switch not {
    case 90...100: print("harf notunuz a")
    case 80..<90: print("harf notunuz b")
    case 70..<80: print("harf notunuz c")
    case 0..<70: print("harf notunuz f")
    default:print("hatalı giriş")
    }
}

File: HarfNotu.py
Not = int(input("Notunuzu girin: "))

if Not <= 100 and Not >= 90:
    print("Harf Notunuz: A")
if Not < 90 and Not >= 80:
    print("Harf Notunuz: B")
if Not < 80 and Not >= 70:
    print("Harf Notunuz: C")
if Not < 70 and Not >= 0:
    print("Notunuz: F")
if Not > 100 or Not < 0:
    print("Hatalı giriş")
